client/mysqltest.cc

```diff
@@ -3811,45 +3811,303 @@ void do_remove_files_wildcard(struct st_command *command)
   DBUG_VOID_RETURN;
 }
 
+/*
+  SYNOPSIS
+  recursive_copy
+  ds_source      - pointer to dynamic string containing source
+                   directory information
+  ds_destination - pointer to dynamic string containing destination
+                   directory information
+  DESCRIPTION
+  Recursive copy of <ds_source> to <ds_destination>
+
+  Note: When symlink appears, copy the target file
+*/
+
+static int recursive_copy(DYNAMIC_STRING *ds_source,
+                          DYNAMIC_STRING *ds_destination) {
```

Comment:
Only flagging once in this review, but curly braces should be on a line on their own.